frontend/src/components/pages/CardDetails/ProjectDetails.jsx

```diff
@@ -117,10 +116,10 @@ export default ({
 
       <Reviews reviewIds={project.projectReviews} reviews={reviews} />
 
-      <ViewContentButton
+      {/* <ViewContentButton // This code prohibits the view content button from always showing when it needs to
```

Comment:
Why is this here?

If the code should be removed then can you please remove it?